nemo/collections/llm/modelopt/quantization/quantizer.py

```diff
@@ -46,6 +46,7 @@
         "int8": mtq.INT8_DEFAULT_CFG,
         "int8_sq": mtq.INT8_SMOOTHQUANT_CFG,
         "fp8": mtq.FP8_DEFAULT_CFG,
+        "block_fp8": mtq.FP8_2D_BLOCKWISE_WEIGHT_ONLY_CFG,
```

Comment:
Question: which ModelOpt version is required to use `mtq.FP8_2D_BLOCKWISE_WEIGHT_ONLY_CFG`?

Reply:
It's going to be in the new release 0.26

Reply:
OK so we wait. Once available would you be willing to add the test that Keval mentioned in the other thread?